domains/s3.is-local.org.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "s3",
+    "domain": "is-local.org",
+    "subdomain": "s3",
```

Comment:
You're going to have to pick a different subdomain, as s3 is an Amazon AWS copyright.

https://aws.amazon.com/trademark-guidelines/

Reply:
pls change it to cdns3